packages/core-components/src/components/Select/Select.tsx

```diff
@@ -190,7 +208,12 @@ export function SelectComponent(props: SelectProps) {
     setOpen(false);
   };
 
-  const handleDelete = (selectedValue: string | number) => () => {
+  const handleDelete = (
+    event: React.MouseEvent,
+    selectedValue: string | number,
+  ) => {
+    event.preventDefault();
```

Comment:
`preventDefault` would not work here.

Reply:
I tried without running this and it was not behaving as expected.